package.json

```diff
@@ -31,6 +31,7 @@
     "prom-client": "^12.0.0",
     "prop-types": "^15.8.1",
     "react": "17.0.2",
+    "react-countdown": "^2.3.2",
```

Comment:
This package isn't well maintained or popular https://github.com/ndresx/react-countdown
It is okay for a prototype but for a real product we should look into better packages